content/knowledge-base/common-Google-Play-Errors.md

```diff
@@ -0,0 +1,82 @@
+
+- `Cannot update a published APK`
+   - If an app has already been sent to a specific track, you cannot re-upload it. - You can change its track or increment the version code.
+   - This error is possible when your version or version code is the same as the apk/aab already uploaded on google play.
+
+- `Your scoped storage permission declaration needs to be updated`
+   - It looks like Google has made updates that required you to declare your [storage permissions](https://developer.android.com/about/versions/11/privacy/storagehttps://www.xda-developers.com/android-11-all-files-access-permission-form/)
+   - You will have to update your app accordingly to be able to publish to Google Play Store.
+
+- `Package not found: com.xxxxx.app.`
+   - The error Package not found means that an application with the package name from the artifact generated during the builds doesn't exist in your Google Play account.
+   - This happens when either, you haven't uploaded the first artifact to your Google Play application manually, or the generated artifact package name has some suffixes or changes and doesn't correspond to the one in Google Play.
+
+- `APK has not been signed with the upload certificate`
+  - Check code signing. Use the same keystore while uploading your artifacts.
+
+- `For uploading an AppBundle you must be enrolled in Play Signing`
+  - As per the error, You need to upload your keystore to [Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?visit_id=637769761748201384-2647523405&rd=1)
+  - Enroll your app into app signing by Google Play to have Google sign the .aab that are generated from the app bundle during installation.
+
+- `You uploaded an APK or Android App Bundle that was signed in debug mode`
+  - You need to sign your APK or Android App Bundle in release mode instead of debug mode.
+  - Set your signing configuration in `build.gradle` in release mode. Refer [this](../code-signing/android-code-signing/#option-2-configure-signing-using-environment-variables) for more information.
+
+- `APKs are not allowed for this application`
+   - APK is not allowed by Google anymore. Instead generate Android App Bundles(.aab). Check [this](https://android-developers.googleblog.com/2021/06/the-future-of-android-app-bundles-is.html) out.
+   - You should use `./gradlew bundleRelease` to generate .aab file. And don't forget to mention the artifact path `app/build/outputs/bundle/**/*.aab` in the `artifacts:` sention in your yaml.
+
+- `Only releases with status draft may be created on draft app` 
+   - `400` error related to the app being in draft status.
+   - The very first version of the app must be added to Google Play manually. 
+   - In case you want to upload the artifacts generated in the build to Google Play as a draft release, select Submit release as draft in workflow editor or `submit_as_draft: true` in yaml configuration.
```

Comment:
```suggestion
   - In case you want to upload the artifacts generated in the build to Google Play as a draft release, select **Submit release as draft** in the workflow editor or `submit_as_draft: true` in your yaml configuration.
```